redwood-widget-compose/src/commonMain/kotlin/app/cash/redwood/widget/compose/ComposeWidgetChildren.kt

```diff
@@ -17,55 +17,39 @@ package app.cash.redwood.widget.compose
 
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.mutableStateListOf
+import app.cash.redwood.LayoutModifier
+import app.cash.redwood.widget.MutableListChildren
+import app.cash.redwood.widget.MutableListChildren.Child
```

Comment:
Not sure this is the right place for this type if it's being exposed by all these varying subtypes. Sure they all use MLC as implementation detail, but it probably belongs next to `Widget.Children` in `Widget`.